tests/test_micropipenv.py

```diff
@@ -195,6 +195,36 @@ def test_install_poetry_vcs(venv):
         check_generated_pipfile_lock(os.path.join(work_dir, "Pipfile.lock"), os.path.join(work_dir, "_Pipfile.lock"))
 
 
+@pytest.mark.online
+def test_install_pipenv_env_vars(venv):
+    """Test installation using enviromental variables."""
+    cmd = [os.path.join(venv.path, BIN_DIR, "python"), micropipenv.__file__, "install", "--method", "pipenv"]
+    with cwd(os.path.join(_DATA_DIR, "install", "pipenv_env_vars")):
+        subprocess.run(cmd, check=True, env=get_updated_env(venv).update({"URL":"https://pypi.org/simple"}))
+        assert str(venv.get_version("daiquiri")) == "2.0.0"
+        assert str(venv.get_version("python-json-logger")) == "0.1.11"
+
+
+@pytest.mark.online
+def test_install_pipenv_env_vars_undefined(venv):
+    """Test installation using enviromental variables without setting them."""
+    cmd = [os.path.join(venv.path, BIN_DIR, "python"), micropipenv.__file__, "install", "--method", "pipenv"]
+    with cwd(os.path.join(_DATA_DIR, "install", "pipenv_env_vars")):
+        subprocess.run(cmd, check=True, env=get_updated_env(venv))
+        assert str(venv.get_version("daiquiri")) == "2.0.0"
+        assert str(venv.get_version("python-json-logger")) == "0.1.11"
+
+
+@pytest.mark.online
+def test_install_pipenv_env_vars_default(venv):
+    """Test installation using default values as URL."""
+    cmd = [os.path.join(venv.path, BIN_DIR, "python"), micropipenv.__file__, "install", "--method", "pipenv"]
+    with cwd(os.path.join(_DATA_DIR, "install", "pipenv_env_vars_default")):
+        subprocess.check_call(cmd, check=True, env=get_updated_env(venv))
```

Comment:
Either you use `subprocess.run` with `check=True` or `subprocess.check_call`. Here you combined both approaches. And because we don't expect the command to fail here, you can also just use `subprocess.run` without `check` because no matter what exit code the command has, we check the installed packages.